server/commands/check.go

```diff
@@ -107,40 +107,15 @@ func (query *CheckQuery) Execute(ctx context.Context, req *openfgapb.CheckReques
 	}, nil
 }
 
-// getTypeDefinitionRelationUsersets validates a tuple and returns the userset corresponding to the "object" and "relation"
-func getTypeDefinitionRelationUsersets(tk *openfgapb.TupleKey, typesys *typesystem.TypeSystem) (*openfgapb.Userset, error) {
-
+// getTypeRelationRewrite validates a tuple and returns the userset corresponding to the "object" and "relation"
```

Comment:
This function doesn't validate the tuple anymore, so let's remove that bit from the godoc comment

Reply:
Ah, good point. Thanks! Updated.